frontend/src/components/landing/BundleBanner.tsx

```diff
@@ -0,0 +1,182 @@
+      alt=""
+      className={styles["main-image"]}
+    />
+  );
+
+  return (
+    <div className={styles["bundle-banner-wrapper"]}>
+      <div className={styles["first-section"]}>
+        <div className={styles["main-img-wrapper"]}>{mainImage}</div>
+        <div className={styles["float-features-wrapper"]}>
+          <FloatingFeatures
+            icon={bundleFloatOne.src}
+            text="bundle-feature-one"
+            position="feature-one"
+            vars={{
+              num_vpn_servers: "400",
+            }}
+          />
+          <FloatingFeatures
+            icon={bundleFloatTwo.src}
+            text="bundle-feature-two"
+            position="feature-two"
+            vars={{
+              num_vpn_countries: "30",
+            }}
+          />
+          <FloatingFeatures
+            icon={bundleFloatThree.src}
+            text="bundle-feature-three"
+            position="feature-three"
+          />
+        </div>
+      </div>
+      {isFlagActive(props.runtimeData, "bundle") &&
+        isBundleAvailableInCountry(props.runtimeData) && (
+          <div className={styles["second-section"]}>
+            <div className={styles["bundle-banner-description"]}>
+              <h2>
+                <Localized
+                  id={"bundle-banner-header"}
+                  elems={{
+                    "vpn-logo": <VpnWordmark />,
+                  }}
+                >
+                  <span className={styles["headline"]} />
+                </Localized>
+              </h2>
+              <h3>{l10n.getString("bundle-banner-subheader")}</h3>
+              <p>{l10n.getString("bundle-banner-body")}</p>
+              <Localized
+                id={"bundle-banner-1-year-plan"}
+                elems={{
+                  b: <b />,
+                }}
+              >
+                <p />
+              </Localized>
+              <div className={styles["pricing-logo-wrapper"]}>
+                <div className={styles["pricing-wrapper"]}>
+                  <Localized
+                    id={"bundle-price-monthly"}
+                    vars={{
+                      monthly_price: getBundlePrice(props.runtimeData),
+                    }}
+                    elems={{
+                      "monthly-price": <p className={styles["price"]} />,
+                    }}
+                  >
+                    <span />
+                  </Localized>
+                  <Localized
+                    id={"bundle-price-save-amount"}
+                    vars={{
+                      savings: "??%", // Design states 50%
+                      old_price: "$??", // Design states $11.99
+                    }}
+                    elems={{
+                      "outdated-price": <s className={styles["price"]} />,
+                    }}
+                  >
+                    <span />
+                  </Localized>
+                </div>
+                <img src={bundleLogo.src} alt="Bundle logo" />
```

Comment:
This alt isn't localised, but also I think not super helpful to visually impaired users? We're explicitly not using "Bundle" wording on the website, so it should probably be something like "Firefox Relay + Mozilla VPN", but if the image doesn't provide any extra information (I think it's the puzzle pieces?), an empty alt might be more appropriate.